oolushola/Banka
# #[165169654] Unit test user login

#### What does this PR do?
Test assertions for user login

#### Description of Task to be completed?
Have the following resource endpoints tested with tdd to check if all assertions are met.

- GET:  /api/v1/auth/register

- POST:  /api/v1/auth/register

#### How should this be manually tested?
After cloning the repo, cd into it and from the terminal RUN 'npm run test' to check the build status of the resource

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
#165169654

#### Screenshots (if appropriate)
NA

#### Questions:
NA
